docs/source/guides/versioning.md

```diff
@@ -5,27 +5,33 @@ description: Everything you need to know about evolving your GraphQL schema
 
 Versioning is a technique to prevent necessary changes from becoming "breaking changes" which affect the existing consumers of an API.  These iterations might be as trivial as renaming a field, or as substantial as refactoring the whole data model.
 
-Developers who have worked with REST APIs in the past have probably recognized various patterns for versioning the API, commonly by using a different URI (e.g. `/api/v1`, `/api/v2`, etc.) or a query parameter (e.g. `?version=1`).
+Developers who have worked with REST APIs in the past have probably recognized various patterns for versioning the API, commonly by using a different URI (e.g. `/api/v1`, `/api/v2`, etc.) or a query parameter (e.g. `?version=1`).  With this technique, an application can easily end up with many different API endpoints over time, and the question of _when_ an API can be deprecated can become problematic.
 
-It’s certainly responsible to consider versioning a GraphQL API in a similar fashion, but GraphQL APIs don't need the same type of versioning and, assuming the correct strategies and precautions are observed, are able to serve various iterations on an API from a single endpoint.
+It might be tempting to version a GraphQL API in the same way, but with the right preparations GraphQL APIs don't require the same versioning technique.  By following the strategies and precautions outlined in this guide and using Apollo tooling that adds clarity to every change, many iterations of an API can be served from a single endpoint.
 
 <h2 id="field-usage">Understanding field usage</h2>
 
-Thanks to its field-based granularity, GraphQL simplifies the process of determining which fields are getting used by a particular client.  By using API management tools like [Apollo Engine](https://www.apollographql.com/engine), it becomes possible to see exactly when a field was last used.
+Rather than returning extensive amounts of data which might not be necessary, GraphQL allows consumers to specify exactly what data they need.  This field-based granularity is valueable and avoids "over-fetching" but by also makes it more difficult to understand what data is currently being used.
 
-The power of such usage metrics becomes incredibly valuable when determining whether it’s possible to safely deprecate fields while updating an API.  Without this information, it’s hard to confidently deprecate a field.
+To improve the understanding of field usage within an API, Apollo Server extends GraphQL with rich tracing data that demonstrates _how_ a GraphQL field is used and _when_ it's safe to change or elminate a field.
```

Comment:
elminate => eliminate

Reply:
should it be `rich tracing data` or `rich data tracing` in the context of this sentence?

Reply:
In this case it should be `rich tracing data` because the GraphQL results are being extended with data which is rich in depth and value!